src/config/dynamic-theme-fixes.config

```diff
@@ -28000,6 +28000,13 @@ img[src$=".svg"]:not([alt="Typst"])
 
 ================================
 
+staging.typst.app
```

Comment:
Adding this would need to have it alphabetically sorted not under typst.app

Reply:
However it would be easier to add alias staging.typst.app as a next line under typst.app as it will be the same fixes used and remove this additional copy of fixes.

Reply:
Example how aliases work:
![obraz](https://github.com/user-attachments/assets/b8cc3c86-885b-46a9-bfcf-9681eb773a9c)